lib/features/home_screen/widgets/home_screen_app_bar.dart

```diff
@@ -33,9 +33,8 @@ class HomeScreenAppBar extends StatelessWidget implements PreferredSizeWidget {
         if (navigation.state.activeDestination ==
             NavigationDestinationData.home)
           IconButton(
-            icon: FaIcon(
-              FontAwesomeIcons.retweet,
-              color: Theme.of(context).colorScheme.onPrimaryContainer,
+            icon: SvgPicture.asset(
```

Comment:
Did you see the new common icons class @TammiLion created? :) Maybe it could be moved directly if we are already changing it

Reply:
I did see that class in the Givt app, i can create a similar one in theis project and add it :)

Reply:
added